### DISCUSSION TITLE
My Portfolio Website Feedback Required

### DISCUSSION THREAD

**Discussion by Vijay263601** [Jay-study-nildana/FrontEndForStudents#26] [Show and tell]
Here @Jay-study-nildana I just completed my portfolio website.. I need your reviews in this....

https://vijay263601.github.io/Vijay-Singh-Majila.github.io/

I also need your suggestions specially in color combination i m not very good in that stuff

**Comment by Jay-study-nildana** [comment]
@Vijay263601 

thanks for putting the question here. 

I will check it out within 2 days (tomorrow, Friday, I will be busy watching movies because of my Movies YouTube Channel), so expect feedback on Saturday.

Thanks. 

Jay.

**Reply by Jay-study-nildana** [reply]
@Vijay263601 

Ah, this is looking very cool. Yes, it looks good. You have linked some projects, and they are all live demos. Nice.

Some Points

1. the website has a very 90s kind of feel to it. especially the background color. You could use something less "colorful". standard colors like white black or grey would be better, I think. again, just an opinion. nothing wrong with it. 
2. along with download CV, you could also have another link, "VIEW CV". you should convert your CV into "MARKDOWN format", which can be uploaded as a repository or even build your CV in HTML. will be easier for people who don't want to download from google drive, but just want to see. Or even a simple PDF version instead of google drive, stored in github and linked can work. 
3. the "more projects" once again, looks very 90s, he he :P 

And, I don't see any "actual" projects. Like 'apps'. Even something like the NASA APOD project, or the open source projects you are contributing with me are counted as actual apps because they consume an API. Along with existing projects, add some projects that are deployed on vercel. 

And, once you feel confident, feel free to mention open source projects that you are working with me as well. and, you can put my details as reference. This might help you get more offers because recruiters will look at my profile, and will be pleased to see that you have an microsoft award winning person who knows about you.

So, there you go. That's my feedback. 

Jay.

**Reply by Vijay263601** [reply]
sure @Jay-study-nildana  I'll update it shortly

**Reply by Vijay263601** [reply]
Now I made some changes and updated as u asked me to @Jay-study-nildana  

https://vijay263601.github.io/Vijay-Singh-Majila.github.io/

**Reply by Jay-study-nildana** [reply]
@Vijay263601 

I will check and update you in 24 hours. As usual, if I forget to update you, please remind me either here or on discord.

Thanks

Jay.

**Comment by Jay-study-nildana** [comment]
@Vijay263601 

here is my feedback.

Part One.

instead of "Open Sourve Project By - Vijayasimha BR"

make it "Open Source Project at Big Babee Small Babee Open Source Organization"

much more professional looking in my opinion.

Part Two

source link, should always go to the github link. Because, tomorrow, I might delete the deployment. Or, put both, deployment and github repo link.

Other Things

Oh, you have put my photo. ah, thanks 🙂 It looks good very good. when you get time, if possible make the above changes.

Thanks.
Jay

**Reply by Vijay263601** [reply]
Changes made :-🙂

**Reply by Jay-study-nildana** [reply]
@Vijay263601 

checking.

**Reply by Jay-study-nildana** [reply]
@Vijay263601 

Excellent.

Now, please remember.

1. Resume and Portfolio needs to be updated regularly. 
1. as you are a fresher, you will be learning more things every month, and of course, as you do more open source with me and other people, more things you will do.
1. so updated with new things, every 2 to 3 months or every month also. 

And, if any HR or company asks they want to speak to me (to give you reference, recommendation) give them my email or linked in or discord, (but please tell them, I don't use phone so there is no whatsapp or speaking on phone non sense)

You can tell people I am your mentor, no problem. 

all the best with your career growth. 

Jay
